src/pragha-utils.c

```diff
@@ -233,30 +233,30 @@ gchar* convert_length_str(gint length)
 	str = g_new0(char, 128);
 	memset(tmp, '\0', 24);
 
-	if (length > 86400) {
+	if (length >= 86400) {
 		days = length/86400;
 		length = length%86400;
 		g_sprintf(tmp, "%d %s, ", days, ngettext("day", "days", days));
 		g_strlcat(str, tmp, 24);
 	}
 
-	if (length > 3600) {
+	if (length >= 3600) {
 		hours = length/3600;
 		length = length%3600;
 		memset(tmp, '\0', 24);
 		g_sprintf(tmp, "%d:", hours);
 		g_strlcat(str, tmp, 24);
 	}
 
-	if (length > 60) {
+	if (length >= 60) {
 		minutes = length/60;
 		length = length%60;
 		memset(tmp, '\0', 24);
 		g_sprintf(tmp, "%02d:", minutes);
 		g_strlcat(str, tmp, 24);
 	}
 	else
-		g_strlcat(str, "00:", 4);
+		g_strlcat(str, "00:", 24);
```

Comment:
> However without the buffer we could get rid of the arbitrary length limitation, and the code would probably be a bit cleaner as well. I'll get a try at it.

Do not worry!, 😉 
It's not something that worries me, just comment that it tries to be overly efficient but I doubt it is... 

With this I am more than happy.. And I can advance to 1.4.. 😄 

Regards,